compiler-plugin/src/main/kotlin/arrow/meta/phases/codegen/ir/IrUtils.kt

```diff
@@ -50,51 +49,48 @@ class IrUtils(
   fun CallableDescriptor.irCall(): IrExpression =
     when (this) {
       is PropertyDescriptor -> {
-        val irField = backendContext.ir.symbols.externalSymbolTable.referenceField(this)
-        irField.owner.correspondingPropertySymbol?.owner?.getter?.symbol?.let { irSimpleFunctionSymbol ->
+        val irField = pluginContext.referenceProperties(this.fqNameSafe).first()
```

Comment:
I started this pull request from the beginning because I've been making changes for 1.3.7, 1.4-M1, 1.4-M2, ... and I found a different approach to solve the incompatibilities. For instance:

```
pluginContext.symbols.externalSymbolTable.referenceField(this)
```

for this sentence. I'll push a new commit with the new approach asap!